selinux/selinux-autorelabel-generator

```diff
@@ -40,7 +40,9 @@ enable_units() {
 
 		[Service]
 		Type=oneshot
+   		ExecStartPre=/bin/sh -c "echo SELinux relabeling ${realdir}"
```

Comment:
Instead of this, you can simply make restorecon more verbose. Then there's even a progress bar

Reply:
There is? -p just writes out a number (in thousands of files) with no context... I'm trying to give users context with so they know which mount is being slow

Reply:
@fcrozat we restorecon individual paths, not the entire OS. 
-p only shows percentage if running against the entire OS.
So yeah, there's a percentage, if it's used precisely in the way we're not using it here :)